tests/main/interfaces-hotplug-basic/task.yaml

```diff
@@ -0,0 +1,28 @@
+summary: Basic test to ensure udev monitor doesn't error out and is enabled.
+
+execute: |
+    # shellcheck source=tests/lib/journalctl.sh
+    . "$TESTSLIB/journalctl.sh"
+
+    # Restart so that we capture early startup messages in the logs
+    systemctl restart snapd.{socket,service}
+
+    # Feature flag doesn't need to be set for udev monitor to start, but make sure
+    # the flag is accepted (TODO: add some checks based on the flag on/off once
+    # hotplug works).
+    echo "Ensure feature flag is enabled"
```

Comment:
This sounds awkward. Can we please drop all references to this flag until it is actually required, and once it is required just add the flag with no comment and no extra checks that it was indeed set?  We don't need to check that set/get works, as that's something for "snap set" tests to figure out. The only thing that might be worth testing in this regard is the fact the feature indeed doesn't work once that's disabled, but that's not done here.